### PR TITLE
Update Ukranian language for GUI 0.12.1

### PR DESCRIPTION
**Pull request by TheFuzzStone**
I am a 99 level noob with GIT, I hope that I did everything right.

### CODE REVIEW

File: translations/monero-core_ua.ts [hunk 1]
```diff
@@ -550,27 +550,27 @@
...
         <source>(only available for local daemons)</source>
-        <translation>(можливо тільки при використанні локальних демонів)</translation>
+        <translation>(доступно тільки при використанні локального демона)</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="74"/>
         <source>Mining with your computer helps strengthen the Monero network. The more that people mine, the harder it is for the network to be attacked, and every little bit helps.&lt;br&gt; &lt;br&gt;Mining also gives you a small chance to earn some Monero. Your computer will create hashes looking for block solutions. If you find a block, you will get the associated reward. Good luck!</source>
-        <translation>Майнінг на вашому комп&apos;ютері допомагає захистити мережу Monero. Чим більше людей майнить, тим складніше атаковати мережу, і кожен майнер вносить свій вклад в цей процес. &lt;br&gt; &lt;br&gt;Майнінг також дає вам невеликий шанс добути декілька монет Monero. Ваш комп&apos;ютер буде шукати хеші для рішення блоків. Якщо ви знайдете блок, то отримаєте винагороду. Удачі!</translation>
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: `<translation>` was removed here, probably by mistake, we need it back or the build will fail

File: translations/monero-core_ua.ts [hunk 2]
```diff
@@ -595,12 +595,12 @@
...
         <location filename="../pages/Mining.qml" line="146"/>
         <source>Couldn&apos;t start mining.&lt;br&gt;</source>
-        <translation>Неможливо запустити майнінг&lt;br&gt;</translation>
+        <translation>Неможливо запустити майнінг.&lt;br&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="148"/>
         <source>Mining is only available on local daemons. Run a local daemon to be able to mine.&lt;br&gt;</source>
-        <translation>Майнінг доступний тільки на локальних демонах. Запустіть локальний демон, щоб майнити</translation>
+        <translation>Майнінг доступний тільки на локальному демоні. Запустіть локальний демон, щоб розпочати майнінг.</translation>
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: please add `&lt;br&gt;` at the end of the string

**Comment by TheFuzzStone** [comment]
@erciccione before create this PR, together with @LvMsterfild we reviewed this translation.

**Comment by LvMsterfild** [comment]
Yeah, already reviewed. Ready to update

File: translations/monero-core_ua.ts (hunk 1 above)
Comment (TheFuzzStone) [reply translations/monero-core_ua.ts]: done

**Comment by erciccione** [comment]
Ok, this is good to me, but need to be squashed, and then opened agains master. I can squash for you if you add me as collaborator to your repo. Then you can open a new PR against master

**Comment by TheFuzzStone** [comment]
@erciccione -- done. Now you are collaborator.

**Comment by luigi1111** [comment]
@erciccione and...rebase. :)

**Comment by erciccione** [comment]
rebased

**Pull request merged**